src/marqo/core/search/hybrid_search.py

```diff
@@ -83,24 +83,36 @@ def search(
 
         marqo_index = index_meta_cache.get_index(config=config, index_name=index_name)
 
-        # TODO: Remove when we support unstructured.
-        if isinstance(marqo_index, UnstructuredMarqoIndex):
-            raise core_exceptions.UnsupportedFeatureError(
-                "Unstructured indexes are not yet supported for hybrid search. "
-                "Please use a structured index.")
-
+        # Version checks (different for structured and unstructured)
         marqo_index_version = marqo_index.parsed_marqo_version()
-        if marqo_index_version < constants.MARQO_HYBRID_SEARCH_MINIMUM_VERSION:
+        if isinstance(marqo_index, StructuredMarqoIndex) and \
+                marqo_index_version < constants.MARQO_STRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION:
```

Comment:
You're comparing a `VersionInfo` with a string. Does this work or do we have a bug in 2.10?

Reply:
I checked the implementation of semver, it supports comparison with a string or a dict. This works as expected. But it would be safer to parse it to a semver. (so even we swap the side it would still work).

Reply:
done